Core/HLE/sceCtrl.cpp

```diff
@@ -188,18 +188,14 @@ void __CtrlButtonUp(u32 buttonBit)
 	ctrlCurrent.buttons &= ~buttonBit;
 }
 
-void __CtrlSetAnalogX(float x, int stick)
+void __CtrlSetAnalogXY(int stick, float x, float y)
 {
-	u8 scaled = clamp_u8((int)ceilf(x * 127.5f + 127.5f));
+	u8 scaledX = clamp_u8((int)ceilf(x * 127.5f + 127.5f));
+	// TODO: We might have too many negations of Y...
```

Comment:
Well, this was initially to have *less* negations: 8c98c7b1415f3094cc7d16bc447ea07e805efdd1 / #287.  Long time ago...

But yeah, I guess it might be best (especially since we expose this through the websocket API) to have this not negate.

-[Unknown]

Reply:
Hah, right... Now I don't know what I think heh. Not expecting to deal with this TODO anytime soon though..